webrender/src/image.rs

```diff
@@ -237,12 +237,12 @@ pub fn compute_tile_range(
     let t0 = point2(
         f32::floor(visible_area.origin.x as f32 * tw),
         f32::floor(visible_area.origin.y as f32 * th),
-    ).cast::<u16>();
+    ).try_cast::<u16>().unwrap_or_else(|| panic!("compute_tile_range bad values {:} {:}", visible_area, tile_size));
```

Comment:
Huh, don't you need `{:?}`? Or then wouldn't it be just `{}`?

Reply:
Yeah. Quite.